scripts/web/blueprints/lecture.py

```
@@ -0,0 +1,584 @@
+    if not chunks_file or not Path(chunks_file).exists():
+        return jsonify(success=False, error="Chunks file not found."), 400
+
+    # Read chunks, optionally skip flagged images
+    new_chunks: list[dict] = []
+    for line in Path(chunks_file).read_text(encoding="utf-8").splitlines():
+        if not line.strip():
+            continue
+        try:
+            chunk = json.loads(line)
+        except Exception:
+            continue
```

Ruff (0.15.11)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)